Userland/Libraries/LibC/grp.cpp

```diff
@@ -159,4 +161,45 @@ int initgroups(const char* user, gid_t extra_gid)
         gids[count++] = extra_gid;
     return setgroups(count, gids);
 }
+
+int putgrent(const struct group* g, FILE* stream)
```

Comment:
```suggestion
int putgrent(const struct group* group, FILE* stream)
```

I know `putpwent` also just calls this `p`, but that's not a good variable name. :)